intelmq/bots/outputs/amqptopic/output.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+import sys
+
+import pika
```

Comment:
Have a look here: https://github.com/certtools/intelmq/blob/master/intelmq/bots/experts/asn_lookup/expert.py#L8

Using this method, the bot will fail at initialization time and thus log the error.